ttnn/cpp/ttnn/operations/pool/generic/device/pool_op.hpp

```diff
@@ -80,5 +84,5 @@ struct MaxPool2D {
 }  // namespace ttnn::operations
 
 namespace ttnn::prim {
-constexpr auto max_pool2d = ttnn::register_operation<"ttnn::prim::max_pool2d", ttnn::operations::pool::MaxPool2D>();
+constexpr auto pool2d = ttnn::register_operation<"ttnn::prim::pool2d", ttnn::operations::pool::Pool2D>();
```

Comment:
I think `max_pool2d` as an operation still makes sense. It called to a pool2d device operation, passing MaxPool2D enum value. Unless @pavlejosipovic @mywoodstock prefer it this way, I think its best to correct this change to retain max_pool2d operation.

Reply:
Ah, nvm, I found the max_pool2d op declaration below.